### PR TITLE
Remove event.original removal processors (integrations B* to C*)

### PR DESCRIPTION
**Pull request by Alphayeeeet**
Please label as enhancement

## Proposed commit message 
Remove event.original removal processors

- [x] Barracuda
- [x] Barracuda Cloudgen FW
- [x] Bitdefender
- ~[ ] Bluecoat~ --> will be reverted (Deprecated)
- [x] Box events
- [x] Carbon Black Cloud
- [x] Cassandra
- [x] Ceph
- [x] Checkpoint
- [x] Cisa Kevs
- [x] Cisco Aironet
- [x] Cisco Asa
- [x] Cisco Duo
- [x] Cisco Ftd
- [x] Ciso Ios
- [x] Cisco Ise
- [x] Cisco Meraki
- [x] Cisco Nexus
- [x] Cisco Secure Email Gateway
- [x] Cisco Secure Endpoint
- [x] Cisco Umbrella
- [x] Citrix ADC
- [x] Citrix WAF
- [x] Cloudflare
- [x] Cloudflare Logpush
- [x] CoreDNS
- [x] Crowdstrike
- [x] Cyberarkpas
- [x] Cybereason
- [x] Cylance

## Related issues
Review #10072 for additional info

### CODE REVIEW

**Comment by Alphayeeeet** [comment]
Bluecoat can be removed. I will not touch it as it is deprecated. Any changes will be reverted

**Comment by andrewkroh** [comment]
Bump it to 8.11.

**Comment by Alphayeeeet** [comment]
Ok changelogs have been published. I hope I did not forget any integrations between B* and C*. If so, please tell me and I will remove the processors there aswell.

**Comment by Alphayeeeet** [comment]
@andrewkroh May I request to run CI? As I have not tested locally (as testing locally is afaik only possible per package (and i need it to run ~30 times)). I will check any failed CI tests.

File: packages/barracuda/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.15.0"
+  changes:
+    - description: "Allow @custom pipeline access to event.original without setting preserve_original_event."
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10417
```
[the remark this answers is not in the document]
Comment (Alphayeeeet) [reply packages/barracuda/changelog.yml:6]: Thanks for mentioning. I will replace it and commit tomorrow. I will also review the test results from CI and comment again.

**Comment by Alphayeeeet** [comment]
@andrewkroh Fixed the PR URL's and hopefully failed CI tests. Could you run CI again to check?

**Comment by Alphayeeeet** [comment]
@andrewkroh Are there some vacation outages on obs-infraobs team or is that a coincidence that PR's are stacking up that much and are not beeing reviewed? Do you have any information in that case?

Also please run CI again, because of resolved merge conflicts. Thank you

**Review by muthu-mps — APPROVED** [review]

**Comment by Alphayeeeet** [comment]
@andrewkroh Please run CI again. I have no clue why it's failing. It says something about I/O timeout when setting up go-environment in cisco_ise:

image

**Pull request merged**